schema/vrs.json

```diff
@@ -496,46 +633,79 @@
             }
          }
       },
-      "Quantity": {
-         "description": "A value representing a multitude or magnitude.",
+      "Numeric": {
```

Comment:
I'd prefer to standardize on nouns as type names as we've done in the remainder of the spec. i.e., Number rather than Numeric.